jupyter/README.md

```diff
@@ -0,0 +1,417 @@
+
+_--_
+
+
+
+This tutorial will demonstrate how to run [cirq](https://cirq.readthedocs.io/en/latest/index.html) on Google Cloud Platform. This tutorial will walk through how to install cirq within a Docker container on a GCE Virtual Machine and view results. You will run simulation both in a Jupyter environment and interactively within the container.
+
+Explaining the concepts of quantum computing is beyond the scope of this tutorial, but many excellent references and texts exist. This website describes [Cirq](https://cirq.readthedocs.io/en/stable/index.html) in detail. Additionally,  the textbook “Quantum Computation and Quantum Information” by Nielsen and Chuang is an excellent reference.
+
+------
+
+
+## **Objectives** {#objectives}
+
+
+
+*   Create a Container Optimized VM
+*   Run Docker container with Jupyter and qsim installed
+*   Run a demo circuit
+
+----
+
+
+## **Costs** {#costs}
+
+This tutorial uses billable components of Google Cloud Platform, including:
+
+
+
+*   Compute Engine
+
+Use the [Pricing Calculator](https://cloud.google.com/products/calculator) to generate a cost estimate based on your projected usage.
+
+
+## **Before you begin** {#before-you-begin}
+
+These are the steps to follow to get started with this tutorial:
+
+
+
+*   Creating a Cloud Platform project
+*   Enabling billing for the project
+*   Enable Google Compute Engine API
+
+For this reference guide, you need a Google Cloud [project](https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy#projects). You can create a new one, or select a project you already created:
+
+
+
+1. Select or create a Google Cloud project.
+
+[GO TO THE PROJECT SELECTOR PAGE](https://pantheon.corp.google.com/projectselector2/home/dashboard)
+
+
+
+2. Enable billing for your project.
+
+[ENABLE BILLING](https://support.google.com/cloud/answer/6293499#enable-billing)
+
+When you finish this tutorial, you can avoid continued billing by deleting the resources you created. See [Cleaning up](#cleaning-up) for more detail.
+
+
+## Creating a GCE VM  {#creating-a-gce-vm}
+
+Once you have a project enabled with a billing account, you will be able to create resources. The key resource required for this project is a Virtual Machine (VM) used to run the qsim Quantum Simulations. 
+
+
+### Use the Cloud Console {#use-the-cloud-console}
+
+Connect to the Cloud Console and make sure your current project is selected. Them click on "Create" for a new VM instance:
+
+
+
+![alt_text](images/image7.png "image_tooltip")
+
+
+
+### Build a Container Optimized VM {#build-a-container-optimized-vm}
+
+To create the VM use the steps in sequence below:
+
+
+
+*   Change the name of the VM to be something meaningful like "qsim-1".
+*   Choose a Region and Zone
+    *   The values are not too important now, but for latency and availability [you may make different choice](https://cloud.google.com/compute/docs/regions-zones).
```

Comment:
This should link specifically to the zone-selector in that doc:
https://cloud.google.com/compute/docs/regions-zones#available

Also, I would rephrase this as "[This page](link) provides details on selecting a region and zone. The values are not too important for now - just choose the zone that's closest to you."